docs/mecsimcalc-library/plotdraw.md

```diff
@@ -0,0 +1,1108 @@
+| **`text_loc`**           | **dict** (optional)  | Dictionary specifying horizontal and vertical alignment of          |
+
+the text. |
+| **`alpha`** | **float** (optional) | Transparency level of the line segment. |
+
+#### Example:
+
+```python
+>>> import matplotlib.pyplot as plt
+>>> start = (100, 200)
+>>> end = (400, 500)
+>>> plot_segment_pixels(start, end, text="Segment", min_spacing=50)
+>>> plt.show()
+```
+
+### plot_annotate_arrow
+
+[**[Source]**](https://github.com/MecSimCalc/MecSimCalc-utils/blob/v0.1.8/mecsimcalc/eng130/plot_draw.py#L551C1-L699C1)
+
+```python
+plot_annotate_arrow(
+    start_point: tuple,
+    trig_angle: float,
+    vec_length: float,
+    text: str = "",
+    min_spacing: float = 150,
+    fontsize: int = 11,
+    text_loc: dict = {"ha": "center", "va": "top"},
+    arrow_properties: dict = {
+        "width": 2,
+        "head_width": 15,
+        "head_length": 15,
+        "fc": "black",
+        "ec": "black"
+    },
+    reverse_arrow: str = "no",
+    text_in_center: str = "no",
+    rev_text: str = "no",
+    alpha: float = 0.8
+) -> tuple
+```
+
+#### Description:
+
+Plots an annotated arrow starting from a given point and extending in a given direction.
+
+#### Arguments:
+
+| Argument               | Type                 | Description                                                                                               |
+| ---------------------- | -------------------- | --------------------------------------------------------------------------------------------------------- |
+| **`start_point`**      | **tuple**            | The starting point of the arrow (x, y).                                                                   |
+| **`trig_angle`**       | **float**            | The angle of the arrow in degrees.                                                                        |
+| **`vec_length`**       | **float**            | The length of the arrow.                                                                                  |
+| **`text`**             | **str** (optional)   | The text to display near the arrow.                                                                       |
+| **`min_spacing`**      | **float** (optional) | Minimum spacing for the text from the end of the arrow.                                                   |
+| **`fontsize`**         | **int** (optional)   | Font size of the text.                                                                                    |
+| **`text_loc`**         | **dict** (optional)  | Dictionary specifying horizontal and vertical alignment of the text.                                      |
+| **`arrow_properties`** | **dict** (optional)  | Properties for the arrow, including width, head_width, head_length, fill color (fc), and edge color (ec). |
+| **`reverse_arrow`**    | **str** (optional)   | Whether to reverse the direction of the arrow. (Default is 'no')                                          |
```

Comment:
Can this not be boolean True/False?

Reply:
It should be, I'll change it in the next version